pkg/bundle/sync.go

```diff
@@ -152,9 +154,16 @@ func (b *bundle) secretBundle(ctx context.Context, ref *trustapi.SourceObjectKey
 	return string(data), nil
 }
 
-func encodeJKS(trustBundle string) ([]byte, error) {
+// encodeJKS creates a binary JKS file from the given PEM-encoded trust bundle and password.
+// Note that the password is not treated securely; JKS files generally seem to expect a password
+// to exist and so we have the option for one.
+func encodeJKS(trustBundle string, password []byte) ([]byte, error) {
```

Comment:
Don't know much about JKS but a cursory look seems as though the passwords should be string.

```suggestion
func encodeJKS(trustBundle string, password string) ([]byte, error) {
```

Reply:
The API we're calling expects `[]byte`: https://pkg.go.dev/github.com/pavlo-v-chernykh/keystore-go/v4#KeyStore.Store

That was the main reason I went with `[]byte`, but it's also easier to zero out a byte slice than it is to reason about clearing the contents of a `string`. That's not massively relevant here though given we're not treating these as meaningful security boundaries.

Reply:
this is still a private function, so it probably is not important,